### ISSUE TITLE
How can we create video out of Images

### ISSUE THREAD

**Issue by omkarsurya** [devhyper/open-video-editor#71]
How can we create video out of Images

Media3 supports this and exo player also supports this, 

can you add this feature, as an enhancement

**Comment by OnyxJet** [comment]
Agreed, we should be able to add images and create videos. Using this we could make slideshows and more.
